index.css

```diff
@@ -211,6 +201,141 @@ body {
     fill: #5da8ef;
 }
 
+/* media-content section */
+
+.quote {
+    margin: 0 auto;
+    width: 610px;
+    padding-bottom: 62px;
+}
+
+.quote__content {
+    opacity: 0.9;
+    color: #333333;
+    font-size: 26px;
+    font-weight: 400;
+    line-height: 38px;
+    width: 610px;
+    display: block;
+    text-align: center;
+}
+
+q:before, q:after{ content: "";}
+
+.quote__author {
+    opacity: 0.9;
+    color: #333333;
+    font-size: 9px;
+    font-weight: 700;
+    line-height: 55px;
+    text-transform: uppercase;
+    letter-spacing: 5px;
+    display: block;
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+.quote::after {
+    display: block;
+    content: "";
+    text-align: center;
+    height: 100px;
+    width: 2px;
+    margin: 0 auto;
+    background-color: #5da8ef;
+}
+
+.gallery {
+    position: relative;
+    height: 592px;
+}
+
+.gallery__image-col-2 {
+    position: absolute;
+    top: 43px;
+    z-index: 1;
+    right: 2px;
+}
+
+.gallery__content {
+    bottom: 13px;
+    position: absolute;
+    left: 9%;
+}
+
+.gallery__play {
+    width: 68px;
+    height: 68px;
+    box-shadow: -2px -2px 35px rgba(0, 0, 0, 0.13);
+    background-color: #5da8ef;
+    border-radius: 50%;
+    position: absolute;
+    z-index: 2;
+    bottom: 40%;
+    right: 27%;
```

Comment:
This looks like a magic centering and will cause troubles during the responsive phase.